src/main/java/core/basesyntax/report/report/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.report.report;
+
+import core.basesyntax.model.FruitOperation;
+import java.util.List;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String DEFAULT_DELIMITER = ",";
+    private static final String HEADER = "fruit" + DEFAULT_DELIMITER + "quantity";
+    private final StringBuilder stringBuilder;
+
+    public ReportGeneratorImpl() {
+        stringBuilder = new StringBuilder();
+        stringBuilder.append(HEADER);
+    }
+
+    @Override
+    public String getReport(List<FruitOperation> store) {
+        for (FruitOperation fruitOperation : store) {
+            stringBuilder.append(System.lineSeparator())
+                    .append(fruitOperation.getOperation())
+                    .append(DEFAULT_DELIMITER)
+                    .append(fruitOperation.getQuantity());
```

Comment:
The `getReport` method is currently appending the operation type instead of the fruit name. You should be appending the fruit name and its quantity instead as per the requirements.